src/main/java/net/pms/util/ImdbUtil.java

```diff
@@ -99,8 +99,8 @@ public boolean accept(Path entry) throws IOException {
 						if (isBlank(entryName)) {
 							continue;
 						}
-						double score = StringUtils.getJaroWinklerDistance(nfoFileName, entryName);
-						if (score >= 0.85) {
+						double score = new JaroWinklerSimilarity().apply(nfoFileName, entryName);
+						if (score >= 0.85) { // threshold to add candidate to map
```

Comment:
@valib does anything more need to be done with these JaroWinkler changes?

Reply:
@valib ?

Reply:
@SubJunk this could be close to 1 e.g. 0.9 because the .nfo filename is obviously the same like the media filename.

Reply:
@valib I think I will move these changes out of this branch and into a new one, since our API feature don't change that code at all.

Reply:
@SubJunk I think that the problem is here https://github.com/UniversalMediaServer/UniversalMediaServer/blob/054725e0275f7ee345c49f7814234cbeece45826/src/main/java/net/pms/util/OpenSubtitle.java#L3812-L3824

It checks the episode but not checks if the the filename is correct. That it can parse not relevant subtitles.